base/save.go

```diff
@@ -199,32 +199,31 @@ func CreateDataset(ctx context.Context, r repo.Repo, streams ioes.IOStreams, ds,
 		Path:      path,
 	}
 
```

Comment:
Just making sure this is correct. If dry-run is true, we still call dsfs.CreateDataset, right? So the blocks are added to ipfs, but we never make a reference to them?

Reply:
yes. Earlier in the codepath we pass down a `MemStore` instead of the IPFS store, which is the source of the (bad for storing) `/map/` prefix in the CID. So we are calling CreateDataset, but on a throwaway repo

Reply:
Oh of course. 👍 great!